builtins.go

```diff
@@ -1211,6 +1264,7 @@ func buildBuiltinMap(builtins []builtin) map[string]evalCallable {
 
 var funcBuiltins = buildBuiltinMap([]builtin{
 	builtinID,
+	&ternaryBuiltin{name: "substr", function: builtinSubstr, parameters: ast.Identifiers{"str", "from", "len"}},
```

Comment:
Could you put it before `splitLimit`  (both the table entry and the implementation)? While the list follows no strict order, it's good to have similar functions together, e.g. string functions next to other strings functions, math functions next to other math functions etc.

Reply:
done